index.html

```diff
@@ -200,6 +252,23 @@ <h3 class="title">Skills</h3>
       </form>

```

Comment:
- [ ] [REQUIRED] Kindly fix your form. the input field width needs to be reduced. Please refer to the [Figma Template](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=39%3A122).
Also, reduce the size of the c-shaped SVG icon as it is currently altering the text.

Your Implementation   |  Figma Design
:-------------------------:|:-------------------------:
![1](https://user-images.githubusercontent.com/58692590/149607945-edcf2b90-4323-442c-987f-87af70155e39.png) | ![2](https://user-images.githubusercontent.com/58692590/149607951-2ef1ce8f-51c6-4696-ba5b-b21326ac546b.png)